src/main/service.js

```diff
@@ -60,7 +60,7 @@ function signIn(req, res, next) {
  */
 function logout(req, res, next) {
     try{
-        application.logout(process.pid);
+        application.logout(req.body);
         req.log.info("user logged out");
         res.send(200, {code:200});
```

Comment:
Shouldn't an updated token be sent? something that expires the current token? 
Or is this done at FE level?

Reply:
For our project needs, it should be done at the FE level as far as I investigated.